### DISCUSSION TITLE
Can't join voice channels

### DISCUSSION THREAD

**Discussion by dx7v2** [stoatchat/discussions#426] [Q&A] [answered]
When I click a voice channel, it doesn't join. Or when I click call button on my homies group. Nothing happens. They experience the same problem. Is this an issue or a bug?

**Comment by Rexogamer** [comment, marked as answer]
For server voice channels you need to click the call button in the top-right corner to start/join a call. As for the group DM, that sounds like a possible bug - we might need to investigate that one further.
